be/src/storage/meta_reader.cpp

```diff
@@ -90,7 +113,7 @@ Status MetaReader::_fill_result_chunk(Chunk* chunk) {
             TypeDescriptor desc;
             desc.type = TYPE_ARRAY;
             desc.children.emplace_back(item_desc);
-            ColumnPtr column = ColumnHelper::create_column(desc, false);
+            ColumnPtr column = ColumnHelper::create_column(desc, true);
```

Comment:
I'm not sure whether if works when grayscale upgrade

Reply:
> I'm not sure whether if works when grayscale upgrade

meta scan is only used when collecting low-cardinality dict, can we disable it when upgrading? @stdpain